echopype/calibrate/api.py

```diff
@@ -34,11 +36,101 @@ def _compute_cal(
         return cal_obj.compute_Sp(waveform_mode=waveform_mode, encode_mode=encode_mode)
 
 
-def compute_Sv(echodata: EchoData, **kwargs):
-    """Compute volume backscattering strength (Sv) from raw data."""
+def compute_Sv(echodata: EchoData, **kwargs) -> xr.Dataset:
+    """
+    Compute volume backscattering strength (Sv) from raw data.
+
+    The calibration routine varies depending on the sonar type.
+    Currently this operation is supported for the following ``sonar_model``:
+    EK60, AZFP, EK80 (see below for detail).
+
+    The EK80 echosounder can be configured to transmit
+    either broadband (``waveform_mode="BB"``) or narrowband (``waveform_mode="CW"``) signals.
+    When transmitting in broadband mode, the returned echoes are
+    encoded as complex samples (``encode_mode="complex"``).
+    When transmitting in narrowband mode, the returned echoes can be encoded
+    either as complex samples or as power/angle combinations in a format similar to
+    those recorded by EK60 echosounders (``encode_mode="power"``).
+
+    The current calibration implemented for EK80 broadband complex data
+    uses band-integrated Sv with the gain computed at the center frequency
+    of the transmit signal.
+
+    Parameters
+    ----------
+    waveform_mode : {"CW", "BB"}, optional
+        Type of transmit waveform:
+
+        - `"CW"` for narrowband transmission,
+          returned echoes recorded either as complex or power/angle samples
+        - `"BB"` for broadband transmission samples,
+          returned echoes recorded as complex samples
+
+        This parameter is only used if `sonar_model` is `"EK60"` or `"EK80"`,
+        and in those cases it must be specified.
```

Comment:
?? Is this saying that for EK60 and EK80 'waveform_mode` must be specified, or `sonar_model`? The sentence is confusing

Reply:
docstrings were overhauled